src/components/Nav/MyPageLink.js

```diff
@@ -1,32 +1,53 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
 import './Nav.scss';
 
 class myPageLink extends Component {
+  logOut = () => {
+    localStorage.removeItem('TOKKEN');
+    this.props.history.push('/');
+  };
   render() {
+    const token = localStorage.getItem('TOKKEN');
     return (
       <div className="myPageLink">
         <ul>
-          {MY_PAGE_LINK.map(el => {
-            const { key, content } = el;
-            return (
-              <li key={key}>
-                <Link to="/">{content}</Link>
-              </li>
-            );
-          })}
+          {token
+            ? NAV_LINK_TOKENIN.map(menu => {
+                const { key, content, url } = menu;
+                return (
+                  <li key={key}>
+                    <Link to={url}>{content}</Link>
+                  </li>
+                );
+              })
+            : NAV_LINK_TOKENOUT.map(menu => {
+                const { key, content, url } = menu;
+                return (
+                  <li key={key}>
+                    <Link to={url}>{content}</Link>
+                  </li>
+                );
+              })}
         </ul>
+        {token ? <button onClick={this.logOut}>로그아웃</button> : null}
```

Comment:
```suggestion
        {token && <button onClick={this.logOut}>로그아웃</button>}
```
true case만 다룬다면 && 써줄 수 있을거 같아요

Reply:
true case만 다룰경우 &&연산자!!! 기억하겠습니다 🔥